quickstart/armv7a7-imx6ull-evk.md

```diff
@@ -63,38 +63,6 @@ You can leave the terminal with the serial port open, and follow the next steps.
 
 The process comes down to a few steps, described below.
 
-### Uploading Phoenix-RTOS loader (plo) to the RAM memory
```

Comment:
As discussed please remove the section about uploading the whole system image to ram, plo is needed.